ciphers/trifid_cipher.py

```diff
@@ -1,8 +1,28 @@
-# https://en.wikipedia.org/wiki/Trifid_cipher
+"""
+The trifid cipher uses a table to fractionate each plaintext letter into a
+trigram,mixes the constituents of the trigrams, and then applies the table
+in reverse to turn these mixed trigrams into ciphertext letters.
+https://en.wikipedia.org/wiki/Trifid_cipher
+"""
+
 from __future__ import annotations
 
 

```

Comment:
Let's define `character_to_number` once outside the functions.

`number_to_character` can be created from `character_to_number` in one line of code.

https://github.com/TheAlgorithms/Python/blob/579937613a6dc7e099b710e3d57767a2fab115ad/ciphers/morse_code.py#L9-L22

Reply:
We can't hardcode the number_to_character for the functions, because the alphabet is a variable in trifid cipher. Unlike the morse code where each letter has its own representation.
But we can do it for the doctests though.